### PR TITLE
Remove frozendict import to be able to work with Python 3.9+

### PR DESCRIPTION
**Pull request by mstuebner**
Remove frozendict import to be able to work with Python 3.9+

### CODE REVIEW

**Comment by yogeshrnaik** [comment]
> Remove frozendict import to be able to work with Python 3.9+

frozendict is getting used in many places. We need to replace it in all these places.
image

**Comment by yogeshrnaik** [comment]
> Remove frozendict import to be able to work with Python 3.9+

Please remove it from `requirements.txt` file as well.
image

**Comment by mstuebner** [comment]
Removed all occourences of frozendict that I could find

File: camunda/utils/log_utils.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import logging
 
-from frozendict import frozendict
 
-
-def log_with_context(message, context=frozendict({}), log_level='info', **kwargs):
+def log_with_context(message, context={}, log_level='info', **kwargs):
```
Comment (yogeshrnaik) [inline camunda/utils/log_utils.py:4]:
```suggestion
def log_with_context(message, context=None, log_level='info', **kwargs):
```

**Review by yogeshrnaik — CHANGES_REQUESTED** [review]
Please check review comment

**Review by yogeshrnaik — APPROVED** [review]

**Pull request merged**